src/kixi/comms/components/kafka.clj

```diff
@@ -322,7 +322,8 @@
   (start [component]
     (if-not (:producer-in-ch component)
       (let [topics (or topics {:command "command" :event "event"})
-            origin (or origin (.. java.net.InetAddress getLocalHost getHostName))
+            origin (or origin (try (.. java.net.InetAddress getLocalHost getHostName)
+                                   (catch Throwable _ "<unknown>")))
```

Comment:
does this result in a reasonable exception being thrown eventually? (string "<unknown>")